packages/sdk-react/src/actions/otpAuth.ts

```diff
@@ -7,7 +7,7 @@ type OtpAuthRequest = {
   otpId: string;
   otpCode: string;
   targetPublicKey: string;
-  sessionLength?: number | undefined;
+  sessionLength?: number | undefined; // Desired expiration time in seconds for the generated API key
```

Comment:
Ah I meant we could add `seconds` to the param name like `sessionLengthSeconds` (similar to `expirationSeconds`). A little verbose, but might be easier for developers when their editor populates params (they might not see the actual code/comment, but should see what params are available).